examples/milp_factory_schedule.jl

```diff
@@ -0,0 +1,190 @@
+#  Copyright 2017, Iain Dunning, Joey Huchette, Miles Lubin, and contributors
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#############################################################################
+# JuMP
+# An algebraic modeling language for Julia
+# See http://github.com/JuliaOpt/JuMP.jl
+#############################################################################
+
+"""
+      example_factory_schedule()
+
+This is a Julia translation of part 5 from "Introduction to to Linear Programming with Python" 
+available at https://github.com/benalexkeen/Introduction-to-linear-programming
+
+For 2 factories (A,B), minimize the cost of production over the course of 12 months
+while meeting monthly demand. Factory B has a planned outage during month 5.
+
+"""
+
+using JuMP,GLPK,Test
```

Comment:
Sorry to be a little pedantic, but please add spaces (here and throughout) after commas, so
```Julia
using JuMP, GLPK, Test
```

Reply:
fixed in latest commit